helm_deploy/prison-staff-hub/templates/_envs.tpl

```diff
@@ -169,5 +169,11 @@ env:
 
   - name: PATHFINDER_ENDPOINT_API_URL
     value: {{ .Values.env.PATHFINDER_ENDPOINT_API_URL | quote }}
+
+  - name: SOC_URL
```

Comment:
env variables look confused here - you've got `SOC_URL`, should it be `SOC_API_URL`?  Is this an `API` url or a front end URL?

Reply:
SOC_URL is both API and front-end :)